frontend/trpc/routes/dividendComputations.ts

```
@@ -0,0 +1,118 @@
+import { z } from "zod";
+import { createRouter } from "@/trpc";
```

ESLint

[error] '@/trpc' imported multiple times.

(import/no-duplicates)